apps/parfait-screener/main.go

```diff
@@ -0,0 +1,301 @@
+// Copyright 2023 Stock Parfait
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"runtime"
+	"sort"
+
+	"github.com/stockparfait/errors"
+	"github.com/stockparfait/iterator"
+	"github.com/stockparfait/logging"
+	"github.com/stockparfait/stockparfait/db"
+	"github.com/stockparfait/stockparfait/message"
+	"github.com/stockparfait/stockparfait/table"
+)
+
+type Column struct {
```

Comment:
Actually, most of this code should live in the top-level `screener` package and simply be used here. Reserve the app code to the necessary app boilerplate.